hack/fetch-latest-operator-manifests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+: "${OPERATOR_DIR:=operator}"
+
+IMAGE="nixery.dev/shell/jq/git/curl/kustomize/gnutar/gzip"
```

Comment:
We should probably pin this to a specific version, to avoid our pipeline potentially breaking when a new image is pushed

Reply:
There is no version for this image, because it is generated on the fly with the latest version of all component. But we should build a proper image for this later.

Reply:
Manifests image generation also uses nixery, so it makes sense to create an ondat-tooling image and use it for these automations.